src/core_plugins/timelion/public/vis/index.js

```diff
@@ -19,7 +19,7 @@ define(function (require) {
       title: 'Timeseries',
       icon: 'fa-clock-o',
       description: 'Create timeseries charts using the timelion expression language. ' +
-        'Perfect for computing and combining timeseries set with functions suchs as derivatives and moving averages',
+        'Perfect for computing and combining timeseries set with functions such as derivatives and moving averages',
```

Comment:
I see your point here. In this case I think "set" is a noun, so should be plural, "sets".